openverse_catalog/dags/database/loader_workflow.py

```diff
@@ -104,12 +104,19 @@
         """
         ),
     )
+
+    media_type_xcom = (
+        "{{ ti.xcom_pull(task_ids='%s', key='media_type') }}"
+        % stage_oldest_tsv_file.task_id
```

Comment:
I always preferred a good old `.format` call than the `%` version when possible. Too many rules to remember with the `%` method.

```suggestion
        "{{{{ ti.xcom_pull(task_ids='{task_id}', key='media_type') }}}}".format(
            task_id=stage_oldest_tsv_file.task_id
        )
```

Or even just an `f` string:

```suggestion
        "{{{{ ti.xcom_pull(task_ids='{stage_oldest_tsv_file.task_id}', key='media_type') }}}}"
```

I suppose some folks don't like the curly brace escaping though?

Reply:
I would prefer fstrings if they work here.

Reply:
I was trying to avoid having to escape the brackets, since it ends up with a lot of `{{{{` :sweat_smile: If y'all prefer it though, I'll change it!